src/test/system_tests/ceph_s3_tests/test_ceph_s3_deploy.sh

```diff
@@ -24,6 +24,11 @@ if [ ! -d $DIRECTORY ]; then
     cd ${DIRECTORY}
     git checkout ${CEPH_TESTS_VERSION}
     echo "Finished Downloading Ceph S3 Tests"
+
+    echo "Manually fixing s3select tests"
```

Comment:
Would you please change it to Chicago Capitalization (as the previous lines in echo are):
`Manually Fixing S3Select Tests`?